index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Welcome to Pontiony</title>
+    </head>
+    <body>
+        <h1>Welcome to Poionfy! Enjoy your stay</h1>
```

Comment:
could u remove Enjoy ... and make welcome string to red

Reply:
sure can do